### PR TITLE
[Cogs OCR] Update vision SDK per latest update from REST sepc

### PR DESCRIPTION
**Pull request by toothache**
Update vision SDK per latest update from REST sepc:
- `generate.ps1` script is not working per #6309, so run code generation for Vision SDK manually;
- `TextRecognitionMode` parameter is removed from `BatchReadFile` OCR operation in the latest version and is thus a breaking change. So bump the major number for Vision SDK;
- Fix UT cases for `BatchReadFile`.

### CODE REVIEW

File: sdk/cognitiveservices/Vision.ComputerVision/src/Generated/ComputerVisionClient.cs [hunk 1]
```diff
@@ -1195,7 +1195,7 @@ public async Task<HttpOperationResponse<DomainModelResults>> AnalyzeImageByDomai
         /// <return>
         /// A response object containing the response body and response headers.
         /// </return>
-        public async Task<HttpOperationResponse<OcrResult>> RecognizePrintedTextWithHttpMessagesAsync(bool detectOrientation, string url, OcrLanguages language = default(OcrLanguages), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<HttpOperationResponse<OcrResult>> RecognizePrintedTextWithHttpMessagesAsync(bool detectOrientation, string url, OcrLanguages? language = default(OcrLanguages?), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```
Comment (toothache) [inline sdk/cognitiveservices/Vision.ComputerVision/src/Generated/ComputerVisionClient.cs]: No swagger change for this API, so it seems that this change is caused by the autorest tool change.

File: sdk/cognitiveservices/Vision.ComputerVision/src/Generated/ComputerVisionClient.cs [hunk 2]
```diff
@@ -3731,7 +3723,7 @@ public async Task<HttpOperationResponse<DomainModelResults>> AnalyzeImageByDomai
         /// <return>
         /// A response object containing the response body and response headers.
         /// </return>
-        public async Task<HttpOperationResponse<OcrResult>> RecognizePrintedTextInStreamWithHttpMessagesAsync(bool detectOrientation, Stream image, OcrLanguages language = default(OcrLanguages), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<HttpOperationResponse<OcrResult>> RecognizePrintedTextInStreamWithHttpMessagesAsync(bool detectOrientation, Stream image, OcrLanguages? language = default(OcrLanguages?), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```
Comment (toothache) [inline sdk/cognitiveservices/Vision.ComputerVision/src/Generated/ComputerVisionClient.cs]: Same for this API.

**Comment by toothache** [comment]
We have another spec change, https://github.com/Azure/azure-rest-api-specs/pull/6056. Do not merge the change until that is merged.

**Comment by toothache** [comment]
@AlexGhiondea , we currently have a S3 issue related with this fix. And we need to fix the bug and release a new sdk before 5.31. Can you help review the change?

File: sdk/cognitiveservices/Vision.ComputerVision/src/Generated/ComputerVisionClient.cs [hunk 3]
```diff
@@ -2254,7 +2253,7 @@ public async Task<HttpOperationResponse<TextOperationResult>> GetTextOperationRe
         /// <return>
         /// A response object containing the response body and response headers.
         /// </return>
-        public async Task<HttpOperationHeaderResponse<BatchReadFileHeaders>> BatchReadFileWithHttpMessagesAsync(string url, TextRecognitionMode mode, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<HttpOperationHeaderResponse<BatchReadFileHeaders>> BatchReadFileWithHttpMessagesAsync(string url, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```
Comment (AlexGhiondea) [inline sdk/cognitiveservices/Vision.ComputerVision/src/Generated/ComputerVisionClient.cs]: Is this a breaking change?

**Comment by AlexGhiondea** [comment]
I am seeing a few API changes (removing parameters) -- are those breaking changes?

File: sdk/cognitiveservices/Vision.ComputerVision/src/Generated/ComputerVisionClient.cs (hunk 3 above)
Comment (toothache) [reply sdk/cognitiveservices/Vision.ComputerVision/src/Generated/ComputerVisionClient.cs]: Yes. It's a breaking change.

**Review by AlexGhiondea — APPROVED** [review]

**Pull request merged**